### ISSUE TITLE
Post-beta enhancements

### ISSUE THREAD

**Issue by mturley** [konveyor/virt-ui#132]
Opening this as a catch-all for anything we come across that we should add after the beta release. I figure we can collect these here for now and then break them out into individual issues when it's time to assign and work on them.

So far I can think of:
* Pre/post migration hooks
  * Top-level page
  * Wizard step
* Ability to specify new names for each migrated VM in the plan wizard
* Ability to specify a different target namespace for each VM in the plan wizard

**Comment by vconzola** [comment]
@mturley Marco should create and prioritize this list. I don't understand why the second bullet would be a priority at all.

**Comment by mturley** [comment]
@vconzola Yeah, I'm assuming we'll discuss and prioritize these later and probably drop some. It's just something @jortel mentioned and I didn't want it to get lost.

**Comment by mperezco** [comment]
Let's put this in the parking lot

**Comment by mturley** [comment]
@mperezco I like that, I'm going to make a `parking-lot` label :)

**Comment by mperezco** [comment]
Thanks a bunch!
